MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Indirect/ISISEnergyTransfer.ui

```diff
@@ -819,6 +819,9 @@
         </item>
         <item>
          <widget class="QCheckBox" name="ckSaveSPE">
+          <property name="enabled">
+           <bool>false</bool>
+          </property>
           <property name="toolTip">
```

Comment:
Seems some of the checkboxes have tooltips, others don't. Can you make this consistent throughout?